showpass/Showpass-E2E-tests
# Add a reusable script to verify successful login

**Description:**
Create a script to verify that a user can successfully log into the portal.

- Navigate to [Log In](https://beta.showpass.com/accounts/login/) 
- Input valid credentials (Email & Password)
- Click ‘Log In’
- Make sure you land on the right page after logging in

**Jira ticket:**
https://showpass.atlassian.net/browse/TA-4

**How to test:**
Run the following test and ensure it passes 'cypress/e2e/login/login-valid-inputs.cy.js'

**Test execution results:**
<img width="1706" alt="LoginTestPassed" src="https://github.com/showpass/Showpass-E2E-tests/assets/138615449/42b9b8ba-6a97-4258-84c3-0fcd7366aee9">
